packages/components/package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "@dxe/petitions-components",
+    "type": "module",
+    "scripts-comment": "Note react-jsx or react-jsxdev can be passed to tsc --jsx flag",
+    "scripts": {
+        "dev": "tsc --watch && (concurrently \"tsc -w\" \"tsc-alias -w\")",
+        "build": "tsc && tsc-alias -p tsconfig.json"
+    },
+    "exports": {
```

Comment:
i'm fine w/ this, but i feel like i usually see people make an index file and export everything from there instead of maintaining it in the package.json.